### DISCUSSION TITLE
Quest Req's for "Ghosts Ahoy"

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#16729] [Ideas]
Just letting yall know that one of the item requirements for the quest is the "Ghostspeak Amulet" however, "Restless Ghost" isnt a pre-req quest for the mission. Some simple googling figured the issue out for me, but i just thought you'd like to know! 

Thanks!

**Comment by Nightfirecat** [comment]
What plugin does this affect?

**Comment by ghost** [comment]
Quest helper!Sent from my iPhoneOn Jun 30, 2023, at 3:06 PM, Jordan ***@***.***> wrote:﻿
What plugin does this affect?

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.

**Comment by ghost** [comment]
Yeah no, I’m done with it. I did my part by saying something was messed up. If you want to actually fix what’s wrong with a plug in on your platform then that’s a you guys move. Not tracking down more ppl to say hey fix this lol. Sent from my iPhoneOn Jun 30, 2023, at 4:50 PM, Jordan ***@***.***> wrote:﻿
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the ? button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the Support menu option.

**Reply by Felanbird** [reply]
You told us a plugin we didn't make is broken, I wouldn't classify that as doing your part.
